on_24.rb

```diff
@@ -0,0 +1,1912 @@
+        verb = input['verb']
+        if %w[get post put patch options delete].exclude?(verb)
+          error("#{verb.upcase} not supported")
+        end
+        path = input['path']
+        data = input.dig('input', 'data') || {}
+        if input['request_type'] == 'multipart'
+          data = data.each_with_object({}) do |(key, val), hash|
+            hash[key] = if val.is_a?(Hash)
+                          [val[:file_content],
+                           val[:content_type],
+                           val[:original_filename]]
+                        else
+                          val
+                        end
+          end
+        end
+        request_headers = input['request_headers']
+          &.each_with_object({}) do |item, hash|
+          hash[item['key']] = item['value']
+        end || {}
+        request = case verb
+                  when 'get'
+                    get(path, data)
+                  when 'post'
+                    if input['request_type'] == 'raw'
+                      post(path).request_body(data)
+                    else
+                      post(path, data)
+                    end
+                  when 'put'
+                    if input['request_type'] == 'raw'
+                      put(path).request_body(data)
+                    else
+                      put(path, data)
+                    end
+                  when 'patch'
+                    if input['request_type'] == 'raw'
+                      patch(path).request_body(data)
+                    else
+                      patch(path, data)
+                    end
+                  when 'options'
+                    options(path, data)
+                  when 'delete'
+                    delete(path, data)
+                  end.headers(request_headers)
+        request = case input['request_type']
+                  when 'url_encoded_form'
+                    request.request_format_www_form_urlencoded
+                  when 'multipart'
+                    request.request_format_multipart_form
+                  else
+                    request
+                  end
+        response =
+          if input['response_type'] == 'raw'
+            request.response_format_raw
+          else
+            request
+          end.
+          after_error_response(/.*/) do |code, body, headers, message|
+            error({ code: code, message: message, body: body,
+                    headers: headers }.
+              to_json)
+          end
+        response.after_response do |_code, res_body, res_headers|
+          {
+            body: res_body ? call('format_response', res_body) : nil,
+            headers: res_headers
+          }
+        end
+      end,
+      output_fields: lambda do |object_definition|
+        object_definition['custom_action_output']
+      end
+    }
+  },
+
+  triggers:
+  {
+    new_record:
+    {
+      title: 'New record',
+      subtitle: 'Triggers immediately when new record created',
```

Comment:
Its not webhook trigger, so it won't triggers immediately. Change it to Triggers when a record is created.